src/main/routes/payment-plan-calculation.ts

```diff
@@ -0,0 +1,47 @@
+import * as express from 'express'
+import * as HttpStatus from 'http-status-codes'
+import * as _ from 'lodash'
+
+import { Paths as AppPaths } from 'paths'
+import { createPaymentPlan } from 'common/paymentPlan'
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(AppPaths.paymentPlanCalculation.uri, (req, res) => {
+
+    const totalAmount = req.query['total-amount']
```

Comment:
Can we please add types here and in other places?

Reply:
👍